### PR TITLE
added Comments section for tutorial.md

### PR DESCRIPTION
**Pull request by lukewilliamboswell**

### CODE REVIEW

**Review by Anton-4 — DISMISSED** [review]
Thanks @lukewilliamboswell :)

**Comment by Anton-4** [comment]
Hmm, not sure why CI isn't being triggered...

File: TUTORIAL.md [hunk 1]
```diff
@@ -1416,7 +1416,7 @@ Comments in `.roc` files begin with `##` and will be included in generated docum
 ##     expect x == 2
 ```
 
-Like [Python](https://www.python.org), [Ruby](https://www.ruby-lang.org), [Perl](https://www.perl.org), and [Elixir](https://elixir-lang.org/), Roc also supports inline comments. They begin with `#` instead of `--`, and can also used to add information that won't be included in documentation.
+Like [Python](https://www.python.org), [Ruby](https://www.ruby-lang.org), [Perl](https://www.perl.org), and [Elixir](https://elixir-lang.org/), Roc also supports inline comments. They begin with `#`, and can be used to add information that won't be included in documentation.
```
Comment (lukewilliamboswell) [inline TUTORIAL.md:1419]: Is this confusing now, with the references to the other languages? Maybe we just remove Python, Ruby, Perl, Elixir altogether? It probably doesn't add much value in this context.

Comment (Anton-4) [reply TUTORIAL.md:1419]: Yes, good idea, can you change that?

**Comment by lukewilliamboswell** [comment]
@Anton-4 should be good to go. 👍

**Review by Anton-4 — APPROVED** [review]

**Pull request merged**